### ISSUE TITLE
Missing commits

### ISSUE THREAD

**Issue by FezzFest** [Opendigitalradio/ODR-EDI2EDI#4]
About a year ago I compiled ODR-EDI2EDI from source from this repository. Today I setup a new Pi with ODR-EDI2EDI, but I noticed there's a difference between the two versions: the older version seems to have more options and seems to be based on newer code than the one I just compiled.

`odr-edi2edi --version` reports `v0.4.0-43-g33f07f5` for the version compiled a year ago, while the current code simply reports `v0.4.0`

The help text is different as well:
```ODR-EDI2EDI v0.4.0-43-g33f07f5 starting up

Usage:
odr-edi2edi [options] -c <source>

Options:
The following options can be given only once:
 -m (merge|switch)         Choose input merging or switching mode. (default: merge)
 --switch-delay <ms>       Set the delay after an input interruption before switching (default: 2000 ms).
 -w <delay>                Keep every ETI frame until TIST is <delay> milliseconds after current system time.
                           Negative delay values are also allowed.
 -C <path to script>       Before starting, run the given script, and only start if it returns 0.
                           This is useful for checking that NTP is properly synchronised
 -f <fec>                  Set the FEC, values 0, 1, 2, 3, 4, 5. 0 disables protection. Default 0
 -i <interleave>           Configure the interleaver with given interleave percentage: 0%% send all fragments at once, 100%% spread over 24ms, >100%% spread and interleave. Default 95%%
 -D                        Dump the EDI to edi.debug file.
 -v                        Increase verbosity (Can be given more than once).
 --align <alignement>      Set the alignment of the TAG Packet (default 8).
 -b <backoff>              Number of milliseconds to backoff after an interruption (default 5000).
 -r <socket_path>          Enable UNIX DGRAM remote control socket and bind to given path
 --version                 Show the version and quit.

The following options can be given several times:
EDI Input options
 -c <host:port>            Add enabled input connecting to given host and port using TCP.
 -F <host:port>            Add disabled input connecting to given host and port using TCP.

EDI Output options
 -p <destination port>     Set the destination port.
 -d <destination ip>       Set the destination ip.
 -s <source port>          Set the source port.
 -S <source ip>            Select the source IP in case we want to use multicast.
 -t <ttl>                  Set the packet's TTL.

Debugging utilities
 --live-stats-port <port>  Send live statistics to UDP 127.0.0.1:PORT. Receive with nc -uklp PORT

 --disable-pft             Disable PFT and send AFPackets. (Use only when sending to localhost)
It is best practice to run this tool under a process supervisor that will restart it automatically.
```

Options such as ' --live-stats-port' are nowhere to be found in the current code?
Is my build based on a branch that was later deleted? What happened?

**Comment by mpbraendli** [comment]
Hi, we've temporarily put publication of the next branch on hold for a while. Are you using the newer features of the version you compiled previously, and if yes, which ones?

**Comment by mpbraendli** [comment]
Please see https://github.com/digris/digris-edi-zmq-bridge
